src/commands/addRemoveComment/addRemoveComment.ts

```diff
@@ -0,0 +1,128 @@
+import {
+  window,
+  ExtensionContext,
+  commands,
+  Range,
+  Selection,
+  Position
+} from 'vscode'
+import { MockedActiveEditor } from '../../types/spec/activeEditor'
+import { isWindows } from '@sasjs/utils'
+
+export class AddRemoveCommentCommand {
+  private commentStartRegExp = /^\/\*\s{0,1}/
+  private commentEndRegExp = /\s{0,1}\*\/$/
+  private comment = '/*  */'
+
+  constructor(private context: ExtensionContext) {}
+
+  initialise = () => {
+    const commentOutLineCommand = commands.registerCommand(
+      'sasjs-for-vscode.addRemoveComment',
+      () => this.addRemoveComment()
+    )
+
+    this.context.subscriptions.push(commentOutLineCommand)
+  }
+
+  private isWrappedWithComments(line: string) {
+    return (
+      this.commentStartRegExp.test(line) && this.commentEndRegExp.test(line)
+    )
+  }
+
+  private addRemoveCommentToLine(line: string) {
+    const commentStartRegExp = /^\/\*\s{0,1}/
+    const commentEndRegExp = /\s{0,1}\*\/$/
+
+    const lineWithoutComments = line
+      .replace(commentStartRegExp, '')
+      .replace(commentEndRegExp, '')
+
+    if (this.isWrappedWithComments(line)) {
+      return lineWithoutComments
+    } else {
+      return `/* ${lineWithoutComments} */`
+    }
+  }
+
+  private addRemoveComment = async (
+    mockedActiveEditor?: MockedActiveEditor // INFO: used for unit tests
+  ) => {
+    const activeEditor = mockedActiveEditor || window.activeTextEditor
+
+    if (activeEditor) {
+      const { start, end } = activeEditor.selection
+      const text = activeEditor.document.getText()
+
+      enum LineEndings {
+        CRLF = `\r\n`,
+        LF = `\n`
+      }
+      const lineEnding = new RegExp(LineEndings.CRLF + `$`).test(text)
+        ? LineEndings.CRLF
+        : LineEndings.LF
+
+      const lines = text.split(lineEnding) || []
+
+      const editedLines = lines
+        .reduce((acc: string[], line: string, i: number) => {
+          if (i >= start.line && i <= end.line) {
+            // INFO: \r has to be removed due to line ending differences between Windows and Unix systems
```

Comment:
Might be better to write as:

// INFO: Temporary removal of carriage returns (added back if relevant in join below)